### PR TITLE
Add scroll to top button

### LINKED ISSUE #302: Back to Top Button
A static back to top button is required so that user at anytime viewing this website can move to the top of website.

### PR DESCRIPTION
**Pull request by im-adithya**
## Proposed changes (including videos or screenshots)
Added scroll to top button

https://user-images.githubusercontent.com/64399555/129247975-6769c9c0-9a4c-4a43-82bc-98ec5a490df7.mp4

## Issue(s)
Fixes #302

### CODE REVIEW

**Comment by im-adithya** [comment]
@Erioldoesdesign please review :D

**Review by Arhell — DISMISSED** [review]

File: _sass/_layout.scss [hunk 1]
```diff
@@ -40,6 +40,32 @@ h4 {
...
+  font-size: 30px;
+  text-align: center;
+  color: $grayscale-dark;
+  background-color: #fff;
+  -webkit-transition: background 0.3s ease-in-out;
+  -moz-transition: background 0.3s ease-in-out;
+  transition: background 0.3s ease-in-out;
+  position: fixed;
+  bottom: 50px;
+  right: 10%;
```
Comment (SaptakS) [inline _sass/_layout.scss:59]: Let's fix this to 50px as well. The % doesn't scale very well in different screen sizes and sometimes it's a little too far inside.

**Review by SaptakS — CHANGES_REQUESTED** [review]

**Comment by Erioldoesdesign** [comment]
@im-adithya there are changes on this PR requested please 🙏

**Comment by im-adithya** [comment]
Done! Sorry for the delay!

**Comment by Erioldoesdesign** [comment]
@SaptakS ready for re-review!

**Review by AliReza1083 — COMMENTED** [review]
I reviewed your codes, It was written nice and clean.
wish you all the best.

**Comment by Erioldoesdesign** [comment]
@SaptakS when you get a chance to look? have changes been made?

**Comment by SaptakS** [comment]
Somehow missed the previous messages. Apologies! I will take a look and re-review.

**Review by SaptakS — APPROVED** [review]

**Review by Erioldoesdesign — APPROVED** [review]
LGTM - Let's merge it :)

**Pull request merged**